### ISSUE TITLE
Ability to override flow parameters using AJAX API

### ISSUE THREAD

**Issue by sathish316** [azkaban/azkaban#375]
There is an AJAX API to execute flow:

http://azkaban.github.io/azkaban/docs/2.5/#api-execute-a-flow

From UI, It's possible to override flow parameters before executing a job.
Is it possible to provide the same feature through AJAX API?

**Comment by josiah14** [comment]
Agreed, update the documentation if you support this.  This issue is a deal-breaker for us if we can't do this.

**Comment by josiah14** [comment]
^ @erwa The line of code you provided does not make it obvious that flow parameters are supported per the definition here:  http://azkaban.github.io/azkaban/docs/latest/#executing-flows (go down to Flow Parameters).  I just see there that the request options are being parsed and that it's obvious that the e-mail options are utilized.

**Comment by josiah14** [comment]
Google Groups has figured this out, it just needs to be added to the docs.

https://groups.google.com/forum/#!topic/azkaban-dev/pUYYpsmNsbk

**Comment by vickyi** [comment]
the doc is: 
https://azkaban.readthedocs.io/en/latest/ajaxApi.html#request-parameters-8

Example Values : flowOverride[failure.email]=test@gmail.com
